jquery-sortable-lists.js

```diff
@@ -687,7 +690,26 @@
 		{
 			li.removeClass( 'sortableListsOpen' ).addClass( 'sortableListsClosed' );
 			li.children( 'ul, ol' ).css( 'display', 'none' );
-			li.children( 'div' ).children( '.sortableListsOpener' ).first().css( 'background-image', 'url(' + setting.opener.open + ')' );
+			setOpenClose( li.children( 'div' ).children( '.sortableListsOpener' ).first(), 'open' );
+		}
+
+		/**
+		 * @desc Handles display of open/close image or html
+		 * @param el
+		 * @param state
+		 */
+		function setOpenClose( el, state )
+		{
+			var value = setting.opener[state];
+
+			if( setting.opener.as === 'img' )
```

Comment:
Use only one condition + else. There are only two options so we don't need to check it two times. 
If you decide to use opener.html it can be only if ( opener.html ) ... else ...
But its on you. I'll merge it after this corrections.

May be I am paranoid but I have a fear that every new condition makes the plugin slower and I am waiting when it breaks down.